src/drive/web/modules/viewer/FilesViewer.jsx

```diff
@@ -143,6 +148,10 @@ class FilesViewer extends Component {
               renderFallbackExtraContent={file => (
                 <Fallback file={file} t={t} />
               )}
+              onlyOfficeProps={{
+                isEnabled: isOnlyOfficeEnabled(),
+                opener: file => router.push(makeOnlyOfficeFileRoute(file, true))
+              }}
               panelInfoProps={{
                 showPanel,
                 PanelContent
```

Comment:
Should we start to have a DriveViewer instead of replicating it in several places? What are the diff between them?

Reply:
Oui on pourrait commencer en effet, y'a un sujet sur ces viewers...
Je propose de faire ça entre deux cartes plutôt que sur cette PR, pour la faire avancer.